src/tests/cpp/fwklib/TaskClient.cpp

```diff
@@ -20,12 +20,12 @@
 #include "fwklib/FwkLog.hpp"
 #include "fwklib/PerfFwk.hpp"
 
-#include <stdio.h>
-#include <stdlib.h>
+#include <cstdio>
+#include <cstdlib>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <errno.h>
-#include <string.h>
+#include <string>
 #include <ctype.h>
```

Comment:
Ah.. I must not have `sed`'d that one since it wasn't in our public headers.  Thanks.